features/support/matchers/expect_uploaded_csv_to_include.rb

```diff
@@ -0,0 +1,27 @@
+require 'csv'
+
+CSV::Converters[:boolean] = lambda do |value|
+  case value.to_s
+  when 'true' then true
+  when 'false' then false
+  else value
+  end
+end
+
+module UploadedCsvMatcher
```

Comment:
If we're following the GitHub Ruby style guide we should keep acronyms like CSV uppercase.